website/content/en/v0.25.0/upgrade-guide.md

```diff
@@ -97,6 +97,9 @@ By adopting this practice we allow our users who are early adopters to test out
 
 # Released Upgrade Notes
 
+## Upgrading to v0.26.0+
```

Comment:
Could you add some info in https://karpenter.sh/preview/concepts/deprovisioning/#pod-eviction